content/docs/dev/concepts/explore-tikv-features/cdc.md

```diff
@@ -8,14 +8,352 @@ menu:
         identifier: RawKV CDC-dev
 ---
 
+This page introduces what's RawKV Change Data Capture and how to use it.
+
 ## RawKV Change Data Capture
 
-*TODO*
+**RawKV Change Data Capture** (*abbr.* **RawKV CDC**) is a feature that providing [Change Data Capture] capability for RawKV, to meet high availability requirements.
+
+Using RawKV CDC, you can build a storage system with **Cross Cluster Replication**, to implement financial-level disaster recovery.
+
+RawKV CDC can also be used for integrating RawKV with other data systems.
+
+To use RawKV CDC, you need to enable [TiKV API V2] and deploy a **TiKV-CDC** cluster. The minimal required version of TiKV is [v6.2.0].
+
+{{< figure
+    src="/img/docs/rawkv-cdc.png"
+    caption="RawKV CDC"
+    number="1" >}}
+
+## TiKV-CDC
+
+**TiKV-CDC** is [TiKV](https://docs.pingcap.com/tidb/dev/tikv-overview)'s change data capture framework. It supports replicating change data to another TiKV cluster.
+
+It forks from [TiCDC](https://github.com/pingcap/tiflow/blob/master/README_TiCDC.md), but focus on NoSQL scenario that uses TiKV as a Key-Value storage.
+
+{{< figure
+    src="/img/docs/rawkv-cdc-arch-simple.png"
+    caption="TiKV-CDC Architecture"
+    number="2" >}}
 
 ## Instruction Manual
 
-*TODO*
+### Deployment
+
+#### Deploy by TiUP
+
+_Note: [TiUP] >= `v1.11.0` is required._
+
+##### Deploy a new TiDB/TiKV cluster including TiKV-CDC
+
+When you deploy a new TiDB/TiKV cluster using TiUP, you can also deploy TiKV-CDC at the same time. You only need to add the `kvcdc_servers` section in the initialization configuration file that TiUP uses to start the TiDB/TiKV cluster. Please refer to the configuration [template](https://github.com/tikv/migration/blob/main/cdc/deployments/tikv-cdc/config-templates/topology.example.yaml).
+
+##### Add TiKV-CDC to an existing TiDB/TiKV cluster
+
+You can also use TiUP to add the TiKV-CDC component to an existing TiDB/TiKV cluster. Take the following procedures:
+
+1. Make sure that the current TiDB/TiKV version >= `6.2.0`.
```

Comment:
May need to remind that tikv need enable `api v2` to deploy `tikv-cdc`.

Reply:
Fixed, PTAL~